static/app/utils/metrics/mri.tsx

```diff
@@ -7,21 +7,28 @@ export const DEFAULT_MRI: MRI = 'c:custom/sentry_metric@none';
 export const DEFAULT_METRIC_ALERT_FIELD = `sum(${DEFAULT_MRI})`;
 
 export function isMRI(mri?: unknown): mri is MRI {
-  return !!parseMRI(mri);
+  try {
+    _parseMRI(mri as string);
+    return true;
+  } catch (e) {
```

Comment:
```suggestion
  } catch  {
```

Reply:
oh, is there no linting rule for that? 😮

Reply:
not sure, but it would be a nice to have